### ISSUE TITLE
implement dynamic vinyl to get sent song details(hardcode)

### ISSUE THREAD

**Issue by kithe-raker** [kithe-raker/gimmesong#65]
**DOD**
- [x] create get vinyl component style function
- [x] create new return data schema for get sent song details function
- [x] implement with get sent song details function
- [x] add vinyl component style to project's asset folder & update vinyl style object

**Comment by kithe-raker** [comment]
The approach Im using is create a vinyl stye object that will contains details for each style's id
```js
const __VinylStyle = {
  background: {
    id: { id: "id", image_url: logo },
  },
  center: {
    id: { id: "id", image_url: logo },
  },
};
```
so there are work to put the all vinyl style component in the web-app's asset folder
in order to make this function work properly.

**Comment by kithe-raker** [comment]
example of returned data schema with implemented dynamic vinyl style
```js
{
    "content": {
        "songId": "64xls1fWEyI",
        "message": "ge",
        "song": {
            "videoId": "64xls1fWEyI",
            "length": "4:32",
            "given": 1,
            "thumbnails": [
                {
                    "placeholder": "https://lh3.googleusercontent.com/99wc0-y6a5H5JzN7HeSJciPoGU6lnFH7laVv1DIqmUjyhM0Y05w2ddUFNxpl4dgWh5rv3e8WdiTTHs-teg=w1-h1-p-fSoften=50,50,05-l90-rw",
                    "width": 60,
                    "url": "https://lh3.googleusercontent.com/99wc0-y6a5H5JzN7HeSJciPoGU6lnFH7laVv1DIqmUjyhM0Y05w2ddUFNxpl4dgWh5rv3e8WdiTTHs-teg=w60-h60-l90-rw",
                    "height": 60
                },
                {
                    "width": 120,
                    "height": 120,
                    "placeholder": "https://lh3.googleusercontent.com/99wc0-y6a5H5JzN7HeSJciPoGU6lnFH7laVv1DIqmUjyhM0Y05w2ddUFNxpl4dgWh5rv3e8WdiTTHs-teg=w1-h1-p-fSoften=50,50,05-l90-rw",
                    "url": "https://lh3.googleusercontent.com/99wc0-y6a5H5JzN7HeSJciPoGU6lnFH7laVv1DIqmUjyhM0Y05w2ddUFNxpl4dgWh5rv3e8WdiTTHs-teg=w120-h120-l90-rw"
                }
            ],
            "lastestGiven": {
                "_seconds": 1669346663,
                "_nanoseconds": 162000000
            },
            "title": "He",
            "artistInfo": {
                "artist": [
                    {
                        "text": "Wintley Phipps",
                        "browseId": "UCMDLBL9Sw4YKlTaX2qbiAKA",
                        "pageType": "MUSIC_PAGE_TYPE_ARTIST"
                    }
                ]
            }
        }
    },
    "receivedAt": {
        "_seconds": 1669346663,
        "_nanoseconds": 162000000
    },
    "recipient": "UkNkxeKWcCQ4bKozikaDN3nkCbj1",
    "played": false,
    "id": "bYEsSOMib9x0QBECguoh",
    "vinyl_style": {
        "background": {
            "id": "id",
            "image_url": "/static/media/gimmesong_logo.1600816a01874bced6f6.png"
        },
        "center": {
            "id": "id",
            "image_url": "/static/media/gimmesong_logo.1600816a01874bced6f6.png"
        }
    }
}
```

**Comment by kithe-raker** [comment]
**_NOTE_**
In the implementation process I haven't implement the get vinyl style details in the `RecentlyAdd` array of `SongRequest` yet

**Comment by kithe-raker** [comment]
> **_NOTE_** In the implementation process [bee2ced](https://github.com/kithe-raker/gimmesong/commit/bee2cedc28ee2b65d91bacd73da8596ee3f77130) I haven't implement the get vinyl style details in the `RecentlyAdd` array of `SongRequest` yet

Implemented in aa8a3498b2491918b2eb7adad61b69bb3fbc4e41
```js
    for (let index = 0; index < results?.contents?.length ?? 0; index++) {
      for (
        let vinlyIndex = 0;
        vinlyIndex < results.contents[index].recentlyAdded?.length ?? 0;
        vinlyIndex++
      ) {
        results.contents[index].recentlyAdded[vinlyIndex].vinyl_style =
          await VinylStyle.getVinylStyleDetails(
            results.contents[index].recentlyAdded[vinlyIndex].vinyl_style
          );
      }
    }
```

But still need to refactor in the future

**Comment by kithe-raker** [comment]
Use 1 default background
and 2 new center emoji(Santa and Present)
for the VinylStyle
